src/components/signup/Signup.react.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Header from '../shared/Header.react.js'
+import SignupContainer from '../../containers/SignupContainer.react.js'
+
+const Signup = () => (
+  <div>
+    <Header />
+    <div className={'contentContainer'}>
```

Comment:
nit: does this need to be a string evaluated? Typically I can write this without the curly braces